tree/30_generic_methods/package_absent.cf

```diff
@@ -24,9 +24,9 @@
 # @parameter_constraint version "allow_empty_string" : true
 # @parameter            architecture Architecture of the package, can be an architecture name  or "default" (defaults to "default")
 # @parameter_constraint architecture "allow_empty_string" : true
-# @parameter            provider Package provider to use, can be "yum", "apt", "zypper", "zypper_pattern", "slackpkg", "pkg", "ips" or "default" for system default package manager (defaults to "default")
+# @parameter            provider Package provider to use, can be "yum", "apt", "zypper", "zypper_pattern", "slackpkg", "pkg", "ips", "nimclient" or "default" for system default package manager (defaults to "default")
 # @parameter_constraint provider "allow_empty_string" : true
-# @parameter_constraint provider "select" : [ "", "default", "yum", "apt", "zypper", "zypper_pattern", "slackpkg", "pkg", "ips" ]
+# @parameter_constraint provider "select" : [ "", "default", "yum", "apt", "zypper", "zypper_pattern", "slackpkg", "pkg", "ips", "nimclient" ]
```

Comment:
Why nimclient and not just nim ?

Reply:
because it a nim client and not a nim server

Reply:
it is a package type, how can a package type be a client or a server ?

Reply:
If you use nim on AIX, you have a server that shares object, and a client that installs it, and the command is literraly "nimclient"